packages/config/src/memory/memory.config.ts

```diff
@@ -176,11 +181,13 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
       } as ConfigTileSet;
       removeUndefined(existing);
       this.put(existing);
+    } else {
+      this.duplicateImagery.push(existing);
```

Comment:
I think this is in the wrong spot, should it not be on line 190?

Reply:
Do you just want to catch the ones that have id replaced?
I thought we want to catch all the ones have duplicated imagery names in configs with different ids.

Reply:
The tileset will always be duplicated as this function will be called, for 2193 imagery AND 3857 imagery

so this check needs to move down into the projection logic.